pixi.toml

```diff
@@ -97,27 +74,28 @@ tbump = ">=6.9.0,<6.10"
 [feature.dev.tasks]
 update-rattler = "cargo upgrade -p rattler -p file_url -p rattler_cache -p rattler_conda_types -p rattler_digest -p rattler_lock -p rattler_networking -p rattler_repodata_gateway -p rattler_shell -p rattler_solve -p rattler_virtual_packages"
 
-
 [feature.lint.dependencies]
 actionlint = ">=1.7.7,<2"
 go-shfmt = ">=3.11.0,<4"
-pre-commit = ">=4.1.0,<5"
+lefthook = ">=1.11.14,<2"
```

Comment:
let's also remove pre-commit-hooks

Reply:
We can just remove those.
They the last two were a bit annoying anyway, and for `yaml` we can use `prettier` or `dprint`